src/expressions.rst

```diff
@@ -3673,7 +3679,7 @@ Infinite Loops
 
 :dp:`fls_p11qw6mtxlda`
 An :t:`infinite loop expression` is a :t:`loop expression` that continues to
-evaluate its :t:`block expression` indefinitely unless :t:`terminated` with a
+evaluate its :t:`loop body` indefinitely unless :t:`terminated` by a
 :t:`break expression` or a :t:`return expression`.
```

Comment:
Hmm, this isn't complete actually, `?` operator and panics can also stop the evaluation

Reply:
As discussed on Zulip, it is better to be concise.